.vsts-ci.yml

```diff
@@ -8,59 +11,70 @@ resources:
     ref: refs/heads/master
 
 variables:
-  ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+  teamName: Roslyn-Project-System
+  ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
     PB_PublishBlobFeedKey:
```

Comment:
Any reason to persist the "PB" prefix on these?

Reply:
It is just how it is passed down to the CI_Build.